hphp/test/tools/import_zend_test.py

```diff
@@ -334,6 +335,9 @@
     '/ext/standard/tests/general_functions/getservbyname_variation9.php',
     '/ext/standard/tests/general_functions/getservbyname_variation10.php',
     '/ext/standard/tests/general_functions/getservbyport_variation1.php',
+
+    # bad assumption about system call effect
+    '/ext/standard/tests/general_functions/proc_nice_basic.php',
```

Comment:
The actual file needs to be moved from `good/` to `flaky/` as well, otherwise this will not take effect until the next time the tests are imported.